cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/web/OAuth20AccessTokenController.java

```diff
@@ -60,7 +60,7 @@ protected ModelAndView handleRequestInternal(final HttpServletRequest request, f
             // resource owner password grant type
             final J2EContext context = new J2EContext(request, response);
             final ProfileManager manager = new ProfileManager(context);
-            final OAuthUserProfile profile = (OAuthUserProfile) manager.get(true);
```

Comment:
Sounds good, but I do see this line below:

```
return profile instanceof OAuthUserProfile
                    && validator.checkParameterExist(request, OAuthConstants.CLIENT_ID)
                    && validator.checkServiceValid(registeredService);
```

If we are always checking for instance, why cast above? or remove the instance checking?